eqcorrscan/core/match_filter.py

```diff
@@ -3550,6 +3570,16 @@ def match_filter(template_names, template_list, st, threshold,
     :param plotdir:
         Path to plotting folder, plots will be output here, defaults to run
         location.
+    :type xcorr_func: str or callable
+    :param xcorr_func:
+        A str of a registered xcorr function or a callable for implementing
+        a custom xcorr function. For more information check out:
+        :func:`eqcorrscan.utils.correlate.register_array_xcorr`
+    :type concurrency: str
+    :param concurrency:
+        The type of concurrency to apply to the xcorr function. Options are
+        'multithread', 'multiprocess', 'concurrent'. For more details see
+        :func:`eqcorrscan.utils.correlate.get_stream_xcorr`
```

Comment:
Can you add a short example of using a different xcorr_func to the default here please?

Reply:
Ok, does it go directly under the :param: in the docstring or at the end in an examples tag? With numpy style docs it would probably be put at the end under Examples section, not sure how you want me to do it here.

Reply:
Yup, you are correct, I hit the wrong line - example should go at the end after all the arguments and returns.  See [here](https://github.com/eqcorrscan/EQcorrscan/blob/master/eqcorrscan/utils/mag_calc.py#L85) for an example of an example in EQcorrscan.